src/components/lightbox/LightBox.tsx

```diff
@@ -1,50 +1,12 @@
-import type { ReactNode } from 'react';
+import {
+  type BaseDialogProps,
+  BaseDialog,
+} from 'src/components/dialog/BaseDialog';
 
-import { BaseDialog } from 'src/components/dialog/_BaseDialog';
-import type { Theme } from 'src/types/Theme';
-
-export type LightBoxProps = {
-  children: ReactNode;
-  className?: string;
-  /** Close when clicking outside dialog (on the backdrop)
-   * @param closeOnBlur
-   * @default true
-   */
-  closeOnBlur?: boolean;
-  /**
-   * Close on pressing 'escape' key
-   * @param closeOnEsc
-   * @default true
-   */
-  closeOnEsc?: boolean;
-  open: boolean;
-  themeOverride?: Theme;
-  width?: number;
-  withBorder?: boolean;
+export type LightBoxProps = BaseDialogProps & {
   onClose: () => void;
 };
 
-export const LightBox = ({
-  children,
-  className,
-  closeOnBlur,
-  closeOnEsc,
-  onClose,
-  open,
-  themeOverride,
-  width,
-  withBorder = false,
-}: LightBoxProps) => (
-  <BaseDialog
-    className={className}
-    closeOnBlur={closeOnBlur}
-    closeOnEsc={closeOnEsc}
-    data-theme={themeOverride}
-    noBorder={!withBorder}
-    onClose={onClose}
-    open={open}
-    width={width}
-  >
-    {children}
-  </BaseDialog>
+export const LightBox = ({ children, ...props }: LightBoxProps) => (
+  <BaseDialog {...props}>{children}</BaseDialog>
```

Comment:
This component is now actually identical to the BaseDialog component. The only difference is the `onClose` prop is required instead of optional.